gqlalchemy/reference/transformations/export.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: export
+title: gqlalchemy.transformations.export
+---
+
+# Abstract
+
+GQLAlchemy currently supports exporting data from Memgraph to NetworkX, DGL and PyG graph format.  
+
+## Transporter
+
+Abstract transporter that provides interface for translating Memgraph data into graph objects like `NetworkX, DGL and PyG`.
+
+## GraphTransporter
+
+`GraphTransporter` is used for converting Memgraph graph to the specific graph type offered by some Python package (PyG, DGL, NX...). Each `GraphTransporter` has a reference to the specific translator object which is used for translating Memgraph data into correct graph type. This translator does all the dirty stuff so check out docs about `DGLTranslator, PyGTranslator and NxTranslator` to find out implementation details.
```

Comment:
```suggestion
`GraphTransporter` is used for converting a Memgraph graph to the specific graph type offered by some Python package (PyG, DGL, NX...). Each `GraphTransporter` has a reference to the specific translator object, which is used for translating Memgraph data into the correct graph type. This translator does all the dirty work, so check out the docs about `DGLTranslator`, `PyGTranslator` and `NxTranslator` to find out implementation details.
```
do we have link about these translators? to link?

Reply:
Yes, added links :+1: